Apps/Contoso.Forms.Puppet/Contoso.Forms.Puppet.UWP/App.xaml.cs

```diff
@@ -19,6 +19,7 @@ sealed partial class App : Application
         /// </summary>
         public App()
         {
+            MobileCenter.SetCountry("United States");
```

Comment:
>MobileCenter.SetCountry("United States"); [](start = 12, length = 41)

Can you add a comment above to briefly explain the reason of explicitly SetCountry at app intialization?